lib/record_store/provider/dnsimple/patch_api_header.rb

```diff
@@ -19,12 +19,17 @@ def execute(method, path, data = nil, options = {})
 
     private
 
-    def rate_limit_sleep(rate_limit_reset, rate_limit_remaining)
+    def rate_limit_sleep(_rate_limit_reset, _rate_limit_remaining)
```

Comment:
As an optional cleanup: Since the method is private to this class, it should be possible to change this method signature to get rid of the unused parameters - there won't be many callsites to change (the callsite in `#execute`, as well as some tests in `test/providers/dnsimple_test.rb`.